CHANGES_NEXT_RELEASE

```diff
@@ -10,3 +10,4 @@
 - Add: NGSIv2 support for Bidirectionality plugin (#527)
 - Add: NGSIv2 support for Multientity plugin (#527)
 - Add: Configuration service is exposed as part of the module.
+- Add: NGSIv2 support for Alias plugin. Casting attributes' values to JSON native types (#583).
```

Comment:
At the end this PR has evolved to a status in which I think it would be more precise to say:

```
- Add: casting attribute's values to JSON native types (activable by config.autocast)
``

The "activable by config.autocast" part only if at the this is done through configuration, of course.

Reply:
Fixed in 5490f80ef568dfbfbafc8aeff3a5978ef6fe7a7a

Reply:
Fixed in 5490f80ef568dfbfbafc8aeff3a5978ef6fe7a7a